createMeanyButelkaObjects.cls

```diff
@@ -0,0 +1,44 @@
+public  class createMeanyButelkaObjects {
+Integer howManyBottleToCreate ;
+public static Integer bottleNumber=0;
+
+public static void createMoreBottleObjects (Integer howManyBottleToCreate){
+
+        List<butelka> bottleObjectsList = new List <butelka>();
+        
+            for(Integer i=0; i< howManyBottleToCreate; i++){
+                Integer newBottleNumber = bottleNumber++;
+                butelka myBootle = new butelka(newBottleNumber, i+5,i+3*3);
+                bottleObjectsList.add(myBootle);
+        
+        }
+        System.debug( bottleObjectsList);
+
+        
+
+    for(butelka bott:bottleObjectsList){
```

Comment:
ZADANIE Z GWIAZDKĄ JAK CI SIE CHCE BO Bardzo sztywna ta pętla w ramach ćwiczenia możesz dorobić metodę, które będzie przyjmować jako parametr ilość butelek i je tworzyć, np podasz 30 butelek i automatycznie pętla przeleje z każdej butelki do każdej następnej, takie przelewanko wodospadowe, że pierwsza dostanie 30 litrów, druga 29 trzecia 28 itd i tak w dół, a jak uruchomisz następną metodą nazwaną przelej, to każda butelka przeleje do następnej 1 litr z Siebie tak, że koniec końców ostatniej się doda jeden a pierwszej odejmie, ale przeleci to sobie przez wszystkie butelki, mozesz do zdebugować